tests/integrationv2/providers.py

```diff
@@ -750,6 +750,8 @@ def cipher_to_priority_str(cipher):
 
     @staticmethod
     def protocol_to_priority_str(protocol):
+        if not protocol:
+            return None
```

Comment:
Can you figure out why we need to do this for protocol_to_priority_str but not any other priority_str method? I don't understand why it wouldn't error if we don't specify a curve, but it errors if we don't specify a protocol.

Reply:
Can the reason be that a set of default elliptic curves/cipher is used if it isn't specified during the TLS handshake ?

Reply:
I don't know, you should find out 😄!